test/options.test.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 
 const { join } = require('path');
-const ValidationError = require('schema-utils/src/ValidationError');
+const {
+  default: ValidationError,
+} = require('schema-utils/dist/ValidationError');
```

Comment:
no need do this, name can be changed and location too, we should use try/catch and use snapshots to validate